blankiety_galantow/core/room.py

```diff
@@ -178,4 +190,13 @@ def get_players_info(self):
     async def send_json_to_all_players(self, json):
         """Send json (given as Python dictionary) to all players"""
         for player in self.players:
-            await player.send_json(json)
\ No newline at end of file
+            await player.send_json(json)
+
+    async def timer_cancel(self):
+        self.game_master.timer.cancel()
+        try:
+            await self.game_master.timer.task
+        except asyncio.CancelledError:
+            pass
+        except KickException:
+            raise
```

Comment:
Nowa linia na końcu pliku, bo linter krzyczy
```suggestion
            raise

```